test/extended/images/helper.go

```diff
@@ -24,8 +24,8 @@ func GetImageLabels(c client.ImageStreamImageInterface, imageRepoName, imageRef
 }
 
 // ModifySourceCode will modify source code in the pod of the application
-// according to the sed script.
-func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, sedScript, file string) error {
+// according to the provided command.
+func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, cmd []string) error {
```

Comment:
I had the string at the beginning but then I've realized that array is more readable and as @mnagy mentioned also more consistent with rest of the code